web/src/stream_popover.ts

```diff
@@ -106,16 +106,21 @@ function build_stream_popover(opts: {elt: HTMLElement; stream_id: number}): void
     const show_go_to_channel_feed =
         user_settings.web_channel_default_view !==
         web_channel_default_view_values.channel_feed.code;
+    const show_go_to_list_of_topics =
+        user_settings.web_channel_default_view !==
+        web_channel_default_view_values.list_of_topics.code;
```

Comment:
We can restrict this to development here as part of merging the code without fully launching it.